adafruit_rgb_display/ili9341.py

```diff
@@ -1,10 +1,15 @@
+"""A simple driver for the ILI9341/ILI9340-based displays."""
+
 try:
     import struct
 except ImportError:
     import ustruct as struct
```

Comment:
This can be removed, CircuitPython always has "struct".

Reply:
Argh, sorry, old CircuitPython still uses ustruct.

Reply:
got ya, ignored